cmd/private_network_create.go

```diff
@@ -22,11 +20,12 @@ type privateNetworkCreateCmd struct {
 
 	Name string `cli-arg:"#"`
 
-	Description string `cli-usage:"Private Network description"`
-	EndIP       string `cli-usage:"managed Private Network range end IP address"`
-	Netmask     string `cli-usage:"managed Private Network netmask"`
-	StartIP     string `cli-usage:"managed Private Network range start IP address"`
-	Zone        string `cli-short:"z" cli-usage:"Private Network zone"`
+	Description string      `cli-usage:"Private Network description"`
+	EndIP       string      `cli-usage:"managed Private Network range end IP address"`
+	Netmask     string      `cli-usage:"managed Private Network netmask"`
+	StartIP     string      `cli-usage:"managed Private Network range start IP address"`
+	Zone        v3.ZoneName `cli-short:"z" cli-usage:"Private Network zone"`
+	Option      []string    `cli-usage:"DHCP network option (format: option1=\"value1 value2\")" cli-flag-multi:"true"`
```

Comment:
Can you give example how would you specify flags with comma separator?

Reply:
i thought about something like `-option dns-servers=1.1.1.1, 2.2.2.2`

Reply:
I think this works much better if we separate flags: instead of `--option dns-servers/ntp-servers/routers/domain-search` we have `--dns-server`, `--ntp-server`, `--router` and `--domain-search` (all `[]string` type so they can be specified multiple times).

With explicit flags we get key validation for free and we can properly document each option (something that is lacking in this PR). Single `--option` makes sense if we dynamically read list of options from API (like we do with DBAAS settings schema), but since possible values are hardcoded in CLI anyway (and we have to update CLI when new option is added) there is no benefit to it.

Reply:
not sure...there is a huge list of options, that potentially could be added...those are just a "small" part of privnet configuration, and putting it to the top level could bring to a mess from the api perspective..in my opinon

Reply:
We still have to document all available options in CLI help so I think UX complexity is unavoidable.

Reply:
is it a bad thing to rely on a customer to consult the openapi spec for this? do we really need to document everything within the cli?

Reply:
CLI does not always transparently map API endpoints to commands so finding the correct documentation in API spec is not as easy for external users as it is for us. Personally I would like to see documentation in CLI, but I'll let others pitch in as well: @sauterp @pierre-emmanuelJ

Reply:
splitted...lets see